docs/modules/setup/pages/configuration.adoc

```diff
@@ -179,3 +179,10 @@ KROKI_EXCALIDRAW_HOST:: Host of the Excalidraw container (default: `127.0.0.1`).
 KROKI_EXCALIDRAW_PORT:: Port of the Excalidraw container (default: `8004`).
 
 NOTE: If you are using the default `docker-compose.yaml` file you can rely on the default values.
+
+== Max URI length
+
+Some diagrams, like Excalidraw, have verbose textual descriptions that will produce long URI.
```

Comment:
@Mogztter I was thinking about adding some information about what length that could be added byt figured out that "best practice" changes dependent on what browsers your aiming for ref [maximum-length-of-a-url-in-different-browsers](https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers/417184#417184). But do you think something about this should be mentioned?

Reply:
Definitely, that's a good idea :+1: 
I remember looking for this information so a `TIP: ` would be extremely valuable.
In a nutshell, if you don't need to navigate to the URL (i.e., you only want to display the image using the `<img>` tag) and you don't need to support IE than you can opt for a max URI length of 10240.
And If you ignore Edge, then you can use >64k but I wouldn't recommend this value (it seems excessive).

Reply:
👍 I'll make a proposal

Reply:
Have a look on the new commit with the tip